docs/docs-content/clusters/edge/edge-host-management/access-console.md

```diff
@@ -0,0 +1,92 @@
+---
+sidebar_label: "Access Edge Management Console"
+title: "Access Edge Management Console"
+description: "Instructions for accessing the Edge Management Console."
+hide_table_of_contents: false
+sidebar_position: 32
+tags: ["edge"]
+---
+
+The Edge Management Console (EMC) allows you to manage Edge hosts in your network locally, upload content such as images
+and packs, and create Edge clusters without connections to a centralized Palette installation.
+
+:::preview
+
+:::
+
+This page guides you through how access the console, and log in, and manage your username and password.
+
+## Log in to Edge Management Console
+
+### Prerequisites
+
+- An Edge host with installed with Edge Installer 4.3 or later using the `airgapped` install mode.
```

Comment:
This prereq confused me. Are we trying to say that the Edge must be on 4.3 or higher AND configured for airgap through the `connect` parameter?

Reply:
I will separate the airgap requirement into a separate bullet. The communication from the Edge team has not been clear what the EMC will do if you attempt to use it with a connected Edge host - and i will clarify this more with the team.